arelle/plugin/validate/NL/rules/fr_nl.py

```diff
@@ -436,6 +451,88 @@ def rule_fr_nl_5_06(
             )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        DISCLOSURE_SYSTEM_NT16,
+        DISCLOSURE_SYSTEM_NT17,
+        DISCLOSURE_SYSTEM_NT18
+    ],
+)
+def rule_fr_nl_5_11(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation] | None:
+    """
+    FR-NL-5.11: Formatting using a limited set of 'escaped XHTML' elements MAY ONLY be included in
+    fact values of facts tagged with concepts of type 'formattedExplanationItemType'.
+    Only certain tags are allowed (see manual). "ul" and "ol" tags have a restriction on the "type" attribute".
+    Any valid tag may have a style attribute, but only certain style properties may be used. (font-family, font-size, color)
+    The manual specifies general readabilty in styling, but that is not practically enforced here.
+    - No font smaller than "3" (font size can be altered in many ways that depend on browser support)
+    - No white text on white background (many potential "color" values are not easily readable)
+    """
+    invalidTypeFacts = []
+    for fact in val.modelXbrl.facts:
+        validType = fact.concept.instanceOfType(pluginData.formattedExplanationItemTypeQn)
+        wrappedContent = f'<div>{fact.textValue}</div>'  # Single root element required, wrap in valid div element
+        invalidTags = set()
+        invalidStyles = set()
+        hasElts = False
+        for elt in etree.fromstring(wrappedContent).iter():
```

Comment:
Does this throw an exception if an arbitrary string fact doesn't parse as XML, e.g., `H&R Block`?

Reply:
I've updated this to yield a warning when a fact of type `formattedExplanationItemType` fails parsing and thus can't have it's XHTML contents validated.